app/src/main/res/layout/topic_info_fragment.xml

```diff
@@ -59,7 +58,7 @@
       <TextView
         android:id="@+id/story_count_text_view"
         android:layout_width="0dp"
-        android:layout_height="wrap_content"
+        android:layout_height="20dp"
```

Comment:
Avoid giving fixed heights for textviews.

Reply:
It has a single line Text-View 
their are three textViews first , second are topic name and number of chapters as they are single line i gave a fixed height and the third textview is content description for that i gave wrapContent!!

Reply:
Mock provided.
Portrait Mock: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/19cfbacf-854c-4c7d-8691-3b3d117e1866/TP-Overview-

Landscape Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/4e4d2aec-fd30-4c20-91ed-daf434f47a89/TP-Info-

Reply:
Thanks for the mock links. 
For height Wrap_content will take size of the textview. So need not have a fixed height in this case. We can give max and minheight as fixed wherever required.